src/main/webapp/styles/forum-style.css

```diff
@@ -146,7 +147,10 @@ i {
 .accepted,
 .accept-button,
 .expand-button,
-.response-form {
+.response-form,
+#search h2,
+#search-input,
+#search-button {
```

Comment:
could you make the search button the search icon like the magnifying glass

Reply:
made it a search icon :)